config/tool_conf.xml.sample

```diff
@@ -113,14 +95,31 @@
     <tool file="maf/maf_by_block_number.xml" />
     <tool file="maf/maf_reverse_complement.xml" />
     <tool file="maf/maf_filter.xml" />
+    <tool file="extract/extract_genomic_dna.xml" />
+  </section>
+  <section id="bxops" name="Operate on Genomic Intervals" version="">
+    <tool file="filters/wiggle_to_simple.xml" />
+    <tool file="stats/aggregate_binned_scores_in_intervals.xml" />
+    <tool file="filters/ucsc_gene_bed_to_exon_bed.xml" />
   </section>
   <section id="stats" name="Statistics">
     <tool file="stats/gsummary.xml" />
     <tool file="filters/uniq.xml" />
   </section>
   <section id="plots" name="Graph/Display Data">
-    <tool file="plotting/bar_chart.xml" />
-    <tool file="plotting/boxplot.xml" />
     <tool file="maf/vcf_to_maf_customtrack.xml" />
   </section>
+  <section id="hgv" name="Phenotype Association">
```

Comment:
I am not sure why we have these enabled on Main but we did not have them in sample, does anybody know?